src/components/partnership/dto/partnership.dto.ts

```diff
@@ -20,8 +20,8 @@ export abstract class SecuredPartnershipAgreementStatus extends SecuredProperty<
   description: SecuredProperty.descriptionFor('a list of partnership types'),
 })
 export abstract class SecuredPartnershipTypes extends SecuredPropertyList<
-  PartnershipType[]
->([PartnershipType]) {}
+  PartnershipType
+>(PartnershipType) {}
```

Comment:
Shouldn't need to explicitly type generic FYI

Reply:
Just fixed this in the last commit